src/lua/httpc.lua

```diff
@@ -241,6 +241,8 @@ end
 --
```

Comment:
Regarding commit message (of 'httpc: implement encoding params' commit):

I think it would be fair to mention that precent encoding is not supported at the moment, because it is quite natural to expect it from this API.

I would even say that the query/www-form parameters support and its percent encoding should be released together. But anyway.